slurm/datadog_checks/slurm/check.py

```diff
@@ -315,6 +325,30 @@ def process_sdiag(self, output):
 
         self.gauge('sdiag.enabled', 1)
 
+    def process_scontrol(self, output):
+        # PID      JOBID    STEPID   LOCALID GLOBALID
+        # 3771     14       batch    0       0
+        # 3772     14       batch    -       -
+
+        if len(self.scontrol_cmd) > 1:
+            base_cmd = self.scontrol_cmd[:-1]
+        else:
+            base_cmd = self.scontrol_cmd
```

Comment:
Could you include a test for this case?

Reply:
I think I might get rid of this if completely. It was from a previous iteration that I was planning. But since I'm building the self.control_cmd, using a constant, it will never be shorter than 2. I'll include a test for just test the base_cmd though.